Peer.Domain/Models/AppConfig.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Peer.Domain.Models
+{
+    public class AppConfig
+    {
+        public GitlabConfig? Gitlab { get; set; }
```

Comment:
Do you have nullable reference types enabled or are these structs?

Reply:
nullable is enabled in Peer.Domain/Peer.Domain.csproj